delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ResourcesSearchParams.scala

```diff
@@ -57,13 +58,28 @@ final case class ResourcesSearchParams(
 
 object ResourcesSearchParams {
 
+  sealed trait TypeOperator extends Product with Serializable
+
+  object TypeOperator {
+    case object And extends TypeOperator
+    case object Or  extends TypeOperator
+
+    implicit val fromStringUnmarshaller: FromStringUnmarshaller[TypeOperator] =
```

Comment:
To delete

Reply:
I don't understand how I could delete this?

Reply:
Sorry, I must have misread something on Friday